src/transactions/page.rs

```diff
@@ -62,7 +65,19 @@ pub fn TransactionSpotlightPage() -> impl IntoView {
         },
     );
 
+    create_effect(move |_| {
```

Comment:
Seems like this is unrelated to the problem to be solved, unless I misunderstand?

Reply:
This sets the memo for the <title> (see test to see how memo ends up in title)